rust/agama-lib/src/questions.rs

```diff
@@ -1,8 +1,12 @@
 //! Data model for Agama questions
 
 use std::collections::HashMap;
+pub mod http_client;
+pub mod model;
 
 /// Basic generic question that fits question without special needs
+/// NOTE: structs below is for dbus usage and holds complete questions data
```

Comment:
```suggestion
/// NOTE: structs below is for D-Bus usage and holds complete questions data
```

Reply:
Please, add a blank line to separate the `NOTE` from the rest (otherwise, everything gets rendered on the same line).

Additionally, I would drop the 'NOTE' prefix. Last but not least, "below" is inaccurate because each struct has a separate page.